### PR TITLE
Alternative docs theme: furo

### PR DESCRIPTION
**Pull request by nqn**
**Screen shots**

Screen Shot 2022-02-01 at 7 07 09 PM

Screen Shot 2022-02-01 at 7 08 27 PM

**Tasks**

 - [ ] Permalink hover issue

### CODE REVIEW

**Comment by nqn** [comment]
Screen Shot 2022-02-01 at 7 33 54 PM

Hover permalinks are still broken

**Comment by moinnadeem** [comment]
This looks great Niklas!

**Comment by bandish-shah** [comment]
This looks great!  Much cleaner than what we have now.

One nitpick, looks like there's some padding or space between the far left edge of the window and the nav sidebar.  Can we make this less exaggerated?  It looks a bit awkward since the logo and search bar are not centered in the nav pane, the symmetry looks off.

**Comment by Averylamp** [comment]
Looks great!  Quite clean

File: docs/source/_static/css/custom.css [hunk 1]
```diff
@@ -99,4 +107,4 @@ body {
 
 footer {
     display: none;
-}
\ No newline at end of file
+} */
```
Comment (ravi-mosaicml) [inline docs/source/_static/css/custom.css:110]: nit: newline

**Review by ravi-mosaicml — APPROVED** [review]
Looks very nice!

**Comment by hanlint** [comment]
@nqn this looks great! can you help resolve some conflicts and get this merged?

**Pull request merged**